### PR TITLE
Added optional parameter 'interval' on CoinsFetchMarketChartParams interface + result type for fetchMarketChart - Coingecko Api

### PR DESCRIPTION
**Pull request by wpoulin**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes:(https://www.coingecko.com/en/api/documentation)
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/coingecko-api/coingecko-api-tests.ts
Comment (domdomegg) [inline types/coingecko-api/coingecko-api-tests.ts:42]: Might be worth keeping a copy of the test without interval to ensure that still works

File: types/coingecko-api/index.d.ts
Comment (domdomegg) [inline types/coingecko-api/index.d.ts:244]: We can actually go a step further for all of these as we know we're comparing a coin against a different currency so we get pairs, e.g.:

```suggestion
    market_caps: [number, number][];
```

File: types/coingecko-api/index.d.ts
Comment (domdomegg) [inline types/coingecko-api/index.d.ts:245]:
```suggestion
    prices: [number, number][];
```

File: types/coingecko-api/index.d.ts
Comment (domdomegg) [inline types/coingecko-api/index.d.ts:246]:
```suggestion
    total_volumes: [number, number][];
```

**Review by domdomegg — APPROVED** [review]

File: types/coingecko-api/index.d.ts
Comment (wpoulin) [reply types/coingecko-api/index.d.ts:244]: it's a great idea but I'm not sold yet whether I prefer "tuples" over arrays. Since it's not critical, I'll keep it as is and will consider updating it later on.

**Review by domdomegg — APPROVED** [review]

**Comment by domdomegg** [comment]
> @wpoulin The CI build failed! Please [review the logs for more information](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/677948705586c8a6c4166c7e1d8fd1f2234ea47a/checks?check_suite_id=4764372278).
> 
> Once you've pushed the fixes, the build will automatically re-run. Thanks!
> 
> **Note: builds which are failing do not end up on the list of PRs for the DT maintainers to review.**

This looks like there was some issue with `git fetch origin master` timing out. Rerunning (either manually, or with an empty commit) should fix.

**Comment by wpoulin** [comment]
Do I have to wait for another review whenever I push new commits even thought you've already approved ?

**Review by domdomegg — APPROVED** [review]

**Comment by domdomegg** [comment]
> Do I have to wait for another review whenever I push new commits even thought you've already approved ?

I believe so

**Comment by wpoulin** [comment]
Ready to merge

**Pull request merged**
